src/test/java/com/epam/tc/hw3/pages/HomePage.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.tc.hw3.pages;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+
+public class HomePage extends BasePage {
+    @FindBy(id = "frame")
+    private WebElement frameWithButton;
+    @FindBy(id = "frame-button")
+    private WebElement buttonFrame;
+    @FindBy(css = ".uui-navigation.nav.navbar-nav.m-l8")
+    private List<WebElement> navigationNavBarElements;
+    @FindBy(css = ".benefit-icon")
+    private List<WebElement> imagesOnIndexPage;
+    @FindBy(css = ".benefit-txt")
+    private List<WebElement> textsOnIndexPage;
+    @FindBy(css = ".sidebar-menu.left")
+    private List<WebElement> itemsOnSideBar;
+    private String navigationElementsTextExpected = "HOME CONTACT FORM SERVICE METALS & COLORS";
+    private List<String> textOnIndexPageExpected = Arrays
+            .asList("To include good practices\nand ideas from successful\nEPAM project",
+                    "To be flexible and\ncustomizable",
+                    "To be multiplatform",
+                    "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…");
+    private String itemsTextOnSideBarExpected = "Home Contact form Service Metals & Colors Elements packs";
+
+    public HomePage() {
+        PageFactory.initElements(webDriver, this);
+    }
+
+    public WebElement getFrameWithButton() {
+        return frameWithButton;
+    }
+
+    public boolean isNavigationBarElementsDisplayed() {
+        Boolean isElementsDisplayed = false;
+        for (WebElement element : navigationNavBarElements
+        ) {
```

Comment:
how do you know which element does not displayed?

Reply:
checks for each element was done